src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -850,13 +851,33 @@ In the Job xml schema it is available as `preciousResult`:
 
 ==== Job Result Map
 
-Job has key-value map associated to it, which is called *Job Result Map*. Each task can contribute to this *Job Result Map*. Consider the following groovy example:
+Job has key-value map associated to it, which is called *Job Result Map*.
+Each task can contribute to this *Job Result Map*. It is write-only map,
+such as task cannot see what other task wrote to this map.
+However, task can rewrite existing values. This map contains String as a key,
```

Comment:
However, task can rewrite existing values. => When several tasks write the same key to the result map, the last write always takes precedence.